README.md

```diff
@@ -5,9 +5,20 @@ This guide reviews the steps required to get thread statistics on an Mbed OS ena
 
 You can build this project with all supported [Mbed OS build tools](https://os.mbed.com/docs/mbed-os/latest/tools/index.html). However, this example project specifically refers to the command-line interface tool [Arm Mbed CLI](https://github.com/ARMmbed/mbed-cli#installing-mbed-cli):
 
-1. Install Mbed CLI.
-1. Clone this repository on your system.
-1. Change the current directory to where the project was cloned.
+1. [Install Mbed CLI](https://os.mbed.com/docs/mbed-os/latest/quick-start/offline-with-mbed-cli.html).
+
+1. Clone this repository on your system, and change the current directory to where the project was cloned and deploy:
+
+    ```bash
+    $ git clone git@github.com:ARMmbed/mbed-os-example-thread-statistics.git && cd mbed-os-example-thread-statistics
+    $ mbed deploy
+    ```
+
+    Alternatively, you can download the example project with Arm Mbed CLI using the `import` subcommand:
+
+    ```bash
+    $ mbed import mbed-os-example-thread-statistics && cd mbed-os-example-thread-statistics
+    ```
```

Comment:
It would be preferrable to replace the second instruction by "Use `mbed import` to clone and deploy this repository on your system"

Reply:
@evedon Done. Please review